pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -75,11 +75,18 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 	}
 	for index, sc := range aiConfig.StorageConfigList {
 		ssPtr := &aiStatus.StorageStatusList[index]
-		location := ssPtr.ActiveFileLocation
-		if location == "" {
-			errStr := "No ActiveFileLocation"
-			log.Error(errStr)
-			return errors.New(errStr)
+		var location string
+
+		switch sc.Target {
+		case "", "disk", "tgtunknown":
+			// Do nothing
+		default:
```

Comment:
What was the bug here?

Reply:
I saw one case of  "No ActiveFileLocation" in Kibana, and that is a spurious error since the code only needs ActiveFileLocation for kernel etc targets. So moved the check inside a switch statement.